src/main/java/org/jabref/logic/exporter/ExporterFactory.java

```diff
@@ -62,6 +62,7 @@ public static ExporterFactory create(CliPreferences preferences) {
         exporters.add(new EmbeddedBibFilePdfExporter(bibDatabaseMode, preferences.getCustomEntryTypesRepository(), fieldPreferences));
         exporters.add(new CffExporter());
         exporters.add(new EndnoteXmlExporter(preferences.getBibEntryPreferences()));
+        exporters.add(new TemplateExporter("MARKDOWN", "md", "markdown", null, StandardFileType.MARKDOWN, layoutPreferences, saveOrder));
```

Comment:
See line 46 - this one should be close to that.

Name is wrong - because `Markdown` can have different facets. Similar to text - can be structured in many formats (Cff, EndnoteXml, ...). Call it "academicpages.github.io" or similar.